examples/getstarted/config/features.js

```diff
@@ -0,0 +1,5 @@
+module.exports = ({ env }) => ({
+  future: {
+    contentReleases: env.bool('STRAPI_FEATURES_FUTURE_CONTENTRELEASES', false),
+  },
+});
```

Comment:
I get the following error:

![Screenshot 2023-12-06 at 11 27 08](https://github.com/strapi/strapi/assets/26598053/af08de80-f767-445a-b3b6-8b582ae755bd)


It seems like it is always trying to read future.contentReleases, which I think is a problem since the future may not be defined right? We could have a config object like this:

```
module.exports = ({ env }) => ({
  dataTransfer: false
  unstsable: {
    reactRouterLoaders: true,
  },
});
```
which I think would still break currently with the same error

Reply:
I don't think we're gonna change the API just cause everyone's finally agreed on it lol. 

I will double check this issue, it might have been forgotten about sorry 🤦🏻‍♀️

Reply:
I'll double check 🤞🏼

Reply:
We now accept partial objects 👍🏼